src/Microsoft.AspNetCore.Routing/DependencyInjection/RoutingServiceCollectionExtensions.cs

```diff
@@ -79,7 +79,7 @@ public static IServiceCollection AddRouting(this IServiceCollection services)
             //
             services.TryAddSingleton<EndpointSelector, EndpointConstraintEndpointSelector>();
             services.TryAddSingleton<EndpointConstraintCache>();
-            services.TryAddEnumerable(ServiceDescriptor.Singleton<MatcherPolicy, HttpMethodEndpointSelectorPolicy>());
+            services.TryAddEnumerable(ServiceDescriptor.Singleton<MatcherPolicy, HttpMethodMatcherPolicy>());
```

Comment:
This class had the wrong name 😆

Reply:
I noticed yesterday the names didn't match :)